test/decl/protocol/objc_protocol_inheriting.swift

```diff
@@ -0,0 +1,20 @@
+// RUN: %target-run-simple-swift | %FileCheck %s
```

Comment:
If we have a test for the metadata, we could probably do without the execution test too. Execution tests have a high test-time cost compared to just compiler tests, and they give weaker cross-platform coverage, since they can only be tested for platforms where CI has executors available. Public CI misses out on iOS/tvOS/watchOS execution tests since it doesn't have devices connected.